tests/test_openvpn.py

```diff
@@ -19,15 +19,19 @@ def test_parse(self):
         self.assertEqual(p.metric, 'static')
 
     def test_json_dict(self):
```

Comment:
this doesn't look right, the two test files look different in number of links and nodes, therefore you'd better have two separate tests and not mix things up to avoid confusion

Reply:
@nemesisdesign I edited the second test file to have exactly the same number of links and nodes.

Reply:
(see my comment above) I think something is not right in the code, the decrease of code coverage is a hint, it should not happen since you have not added any new line of code, so I think some lines of code that were tested before are not being tested now.
Could you try to do as suggested and ensure both tests have their own test function and are really executed?